android/main/java/com/couchbase/lite/internal/CouchbaseLiteInternal.java

```diff
@@ -60,6 +60,7 @@ private CouchbaseLiteInternal() { }
     public static final String PLATFORM = "CBL-ANDROID";
 
     public static final String SCRATCH_DIR_NAME = "CouchbaseLiteTemp";
+    public static final String VS_EXTENSION_NAME = "CouchbaseLiteVectorSearch.so";
```

Comment:
Probably not going to be used?

Reply:
maybe not, now.   It was when I was copying it out of assets.

Reply:
Definitely used, now